db/compaction_iterator.cc

```diff
@@ -1,5 +1,3 @@
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file. See the AUTHORS file for names of contributors.
```

Comment:
But this file is not in LevelDB. I can't go back and track whether any code was copied from there. It wasn't clear to me with a quick peek.